tornjak-frontend/src/components/types.ts

```diff
@@ -94,4 +94,13 @@ export interface TornjakServerInfo {
 export interface ServerInfo {
   trustDomain: string; // Trust domain of server
   nodeAttestorPlugin: string; // Node Attestor Plugin of server
+}
+
+// tornjak 
+export interface StringLabels {
```

Comment:
type name used in context, should be related to its use and not so much its description. An alias of this should be instantiated with using the specific type for example

i.e. like in go lang

```
type AgentLabels StringLabels
type ClusterLabels StringLabels

type StringLabels struct {...}
```

either that, or maybe the struct is not that useful and should be rethinked to just be a string.

Reply:
Makes sense. Created alias for each.